src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx

```diff
@@ -1119,20 +1119,20 @@ The following instructions summarize how to increase these limits if you want to
     apt-cache showpkg fluent-bit
     ```
 
-    To upgrade to a particular Fluent Bit version, just run (in the following examples, version `2.0.8` is used):
+    To upgrade to a particular Fluent Bit version, just run (in the following examples, version `2.2.2` is used):
 
     RPM:
 
     ```bash
     # Remove command only required when downgrading to a previous version
     sudo yum remove fluent-bit
-    sudo yum install fluent-bit-2.0.8-1
+    sudo yum install fluent-bit-2.2.2-1
```

Comment:
@luckslovez could you clarify what you mean by the "-1": fluent-bit-2.2.2-1?

Reply:
We follow the same naming convention as the official fluent-bit packages, see [centos9](https://packages.fluentbit.io/centos/9/) for instance.